### ISSUE TITLE
Linux Runtime 'DOTNETCORE|3.1' is not supported.

### ISSUE THREAD

**Issue by TPIvan** [Azure/azure-cli#13690]
## Describe the bug

**Command Name**
`az webapp create`

**Errors:**
```
Linux Runtime 'DOTNETCORE|3.1' is not supported.Please invoke 'list-runtimes' to cross check
```

## To Reproduce:
I have been working in azure portal Cloud shell and trying to create a Web App from command line
az group create --name azlinux --location "west europe"
az plan create --name azplan --resource-group azlinux --is-linux --sku F1
az webapp create --name "azwebapplinux" --resource-group azlinux --plan azplan --deployment-local-git --runtime "DOTNETCORE|3.1"

when I used --runtime "DOTNETCORE|2.1" it was created succesfully
in web GUI is 3.1 version listed and an app was succesfully created
list-runtime did not listed neither  DOTNETCORE|3.1 nor DOTNETCORE|2.1 (or any other core version)

## Expected Behavior
creating an app  with name azwebapplinux
## Environment Summary
```
Linux-4.15.0-1082-azure-x86_64-with-debian-stretch-sid
Python 3.6.5
Installer: DEB

azure-cli 2.5.1 *

```
## Additional Context
command-modules-nspkg              2.0.3
core                               2.5.1 *
nspkg                              3.0.4
telemetry                          1.0.4

Python location '/opt/az/bin/python3'
Extensions directory '/home/ivan/.azure/cliextensions'

Python (Linux) 3.6.5 (default, Apr 30 2020, 06:22:39)
[GCC 5.4.0 20160609]

Legal docs and information: aka.ms/AzureCliLegal

You have 2 updates available. Consider updating your CLI installation with 'sudo apt-get update && sudo apt-get install --only-upgrade -y azure-cli'. Detailed instructions can be found at https://aka.ms/doc/UpdateAzureCliApt

**Comment by yonzhan** [comment]
webapp

**Comment by panchagnula** [comment]
Should be fixed by change #13688

**Comment by yonzhan** [comment]
move to S171
